src/commands/folding.rs

```diff
@@ -0,0 +1,70 @@
+/*
+ *   Copyright (c) 2020 Owen Salter <owen@devosmium.xyz>
+ *   All rights reserved.
+ */
+
+use crate::prelude::*;
+extern crate reqwest;
+use std::io::Read;
+use serenity::{
+    http::GuildPagination,
+    model::{
+        guild::PartialGuild,
+        id::GuildId,
+        invite::{Invite, RichInvite},
+    },
+};
+
+#[command]
+#[description = "Provides F@H Leaderboard for the DSC F@H Team."]
+pub fn fahleaderboard(ctx: &mut Context, msg: &Message) -> CommandResult {
+    msg.channel_id.send_message(&ctx.http, |m| {
```

Comment:
should have an `if let` block here to handle a message error (see other commands for an example)